main/dhcp/APKBUILD

```diff
@@ -8,6 +8,9 @@ url="https://www.isc.org/"
 arch="all"
 license="ISC"
 depends=
+# busybox provided features are good enough for these
+#depends_dhclient_optional="bash iputils net-tools sed util-linux"
+depends_dhclient="coreutils iproute2 run-parts $depends_dhclient_optional"
 pkgusers="dhcp"
 pkggroups="dhcp"
 makedepends="perl linux-headers file"
```

Comment:
we should also have depends_dhclient in make depends to make sure those are built before `dhcp`.
````sh
makedepends="$depends_dhclient ...."
````